packages/fxa-graphql-api/package.json

```diff
@@ -34,6 +34,7 @@
     "apollo-server": "^2.13.1",
     "apollo-server-express": "^2.13.1",
     "convict": "^6.0.0",
+    "fxa-js-client": "^1.0.25",
```

Comment:
FYI there's a typescript fxa-js-client WIP that's in content-server for now, but should be available soon as a subpackage

Reply:
Yep, waiting for yarn change to land before trying to bring that in.